pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -177,7 +177,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   doCheck = false;
 
   passthru = {
-    tests = {
+    tests = lib.optionalAttrs stdenv.buildPlatform.isDarwin {
```

Comment:
nit: is buildPlatform right? i feel like if we ever got the ability to cross build from linux -> darwin this check would be doing the wrong thing. probably doesn't matter since we can't do that.

Reply:
I am not sure why I put build here, sorry.